src/ui/GenioWindow.cpp

```diff
@@ -3584,6 +3591,9 @@ GenioWindow::_ProjectFolderClose(ProjectFolder *project)
 	if (project == nullptr)
 		return;
 
+	printf("Chiudo il progetto: %s\n", project->Name().String());
```

Comment:
shall we remove it or turn into a LogTrace in english?

Reply:
Ops.. we can remove it.. but this reminds me something about closing a project: while in the context menu the action is referred to the selected item, in the menu bar the action is referred to the Active project.. Maybe it's just me but I took I while to understand it.. I would suggest to rename the menu bar one as 'Close active project" or similar..

Reply:
agree, it sounds sensible.